index.js

```diff
@@ -109,6 +129,21 @@ module.exports = (opts = {}) => {
         }
       }
     },
+    DeclarationExit: (declaration, { postcss }) => {
+      if (!declaration.value.startsWith('light-dark(')) return
+
+      let matches = [...declaration.value.matchAll(LIGHT_DARK)][0]
```

Comment:
Can we have multiple `light-dark()` in the value?

Reply:
indeed, we might have something like `radial-gradient()` which can include multiple `light-dark()`

we can also have `light-dark()` inside strings, in which case we should not touch it